cpu/pmqa.py.data/pmqa.yaml

```diff
@@ -0,0 +1,7 @@
+#Usage: run_arg can  any of one from below list
```

Comment:
Anyway, just an option, case you want to make simpler test and enjoy the multiplex.